src/Neo/Plugins/Plugin.cs

```diff
@@ -229,7 +247,46 @@ protected internal virtual void OnSystemLoaded(NeoSystem system)
         /// <returns><see langword="true"/> if the <paramref name="message"/> is handled by a plugin; otherwise, <see langword="false"/>.</returns>
         public static bool SendMessage(object message)
         {
-            return Plugins.Any(plugin => plugin.OnMessage(message));
+            foreach (var plugin in Plugins)
+            {
+                if (plugin.IsStopped)
+                {
+                    continue;
+                }
+
+                bool result;
+                try
+                {
+                    result = plugin.OnMessage(message);
+                }
+                catch (Exception ex)
+                {
+                    Utility.Log(nameof(Plugin), LogLevel.Error, ex);
+
+                    switch (plugin.ExceptionPolicy)
+                    {
+                        case UnhandledExceptionPolicy.StopNode:
+                            throw;
+                        case UnhandledExceptionPolicy.StopPlugin:
+                            plugin.IsStopped = true;
+                            break;
+                        case UnhandledExceptionPolicy.Ignore:
+                            break;
```

Comment:
we should log it

Reply:
logged by `Utility.Log(nameof(Plugin), LogLevel.Error, ex);` already